packages/forma-36-react-components/src/components/RadioButtonField/README.mdx

```diff
@@ -0,0 +1,71 @@
+Use RadioButtonField to give users the possibility to select only one option from a number of choices. Radio is generally displayed in a radio group. It contains a radio input with its label. If needed the component can display an error message and hint message. For a multiple-choice, please consider using [ChecboxField component](https://f36.contentful.com/components/checkbox-field/).
```

Comment:
> Radio is generally displayed in a radio group

Is there a word missing here?

Reply:
I actually got to the conclusion, that this sentence is not really needed.  It is explained in "How to use" section.